crates/rolldown_common/src/types/entry_point.rs

```diff
@@ -12,7 +12,7 @@ pub struct EntryPoint {
   /// emitted chunk specified filename, used to generate chunk filename
   pub file_name: Option<ArcStr>,
   /// which stmts create this entry point
-  pub related_stmt_infos: Vec<(ModuleIdx, StmtInfoIdx)>,
+  pub related_stmt_infos: Vec<(ModuleIdx, StmtInfoIdx, ImportRecordIdx)>,
```

Comment:
Recording `ImportRecordIdx`, in the future pr, we will support none top-level `ImportExpr` elimination, so that we need to know the `ImportRecordIdx`